tests/helpers.test.ts

```diff
@@ -1,12 +1,15 @@
-import { 
-    validateExpectedTranscription, 
-    createTestData, 
-    parseTextFileContent, 
-    cleanExpectedTranscription,
-    ITestResult,
+import * as path from 'path';
+
+import {
+    createTestData,
+    parseTextFileContent,
     calculateSER
 } from '../src/helpers';
-import * as path from 'path';
+
+import { TranscriptionAnalysisService } from '../src/TranscriptionAnalysisService';
+import { ITestResult } from '../src/types';
```

Comment:
For Consistency, this interface is not implemented by a class, The 'I' should be excluded.

Reply:
Fixed with 5541ae6